### PR TITLE
KP-9134 Add bulk modification tools CMDI metadata

### PR DESCRIPTION
**Pull request by aajarven**
Add the following bulk modification tools:
- based on first and last name, add given affiliation XML to each matching person without affiliation
- switch FIN-CLARIN recorded as person to an organization (when the role allows organization without a person)
- switch Language Bank of Finland recorded as person to an organization (when the role allows organization without a person)

Active modifications are chosen using command line flags. See `python update_cmdi.py --help` for more information. Assuming that `conf/affiliations.json` is populated, the full suite of currently implemented modifications can be run by
```
python update_cmdi.py fake-session-id FIN-CLARIN -v --dry-run \
  --add-affiliations-from conf/affiliations.json \
  --language-bank-to-organization \
  --finclarin-to-organization
```

At the moment, uploading records is not yet possible: it should only be added when we are at least somewhat sure that the modifications are what we want to prevent accidents.

### CODE REVIEW

File: modifiers/lb_modifiers.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                    <url>http://www.helsinki.fi/fin-clarin</url>
+                    <address>PO Box 24 (Unioninkatu 40)</address>
+                    <zipCode>00014</zipCode>
+                    <city>University of Helsinki</city>
+                    <country>Finland</country>
+                </communicationInfo>
+            </organizationInfo>
+            """
+        super().__init__(
+            person_surname="FIN-CLARIN", organization_info_str=organization_info_str
```
Comment (mmatthiesencsc) [inline modifiers/lb_modifiers.py:102]: To be consistent with lines 132/133:
```suggestion
            person_surname="FIN-CLARIN",
            organization_info_str=organization_info_str,
```

File: modifiers/lb_modifiers.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+    Persons are matched based on name (first name and surname) and not having an
+    organization yet.
+    """
+
+    def __init__(self, first_name, surname, organization_info_str):
+        """
+        Create a new modifier for a specific person
+
+        :paran first_name: First name of the edited person
```
Comment (mmatthiesencsc) [inline modifiers/lb_modifiers.py]:
```suggestion
        :param first_name: First name of the edited person
```

File: conf/example_affiliations.json
Comment (mmatthiesencsc) [inline conf/example_affiliations.json:1]: I am unsure about the context of  this file, is it used for testing?

**Review by mmatthiesencsc — CHANGES_REQUESTED** [review]
This code looks very good and extensible. Can't wait to see it in action. I added a few comments/corrections.

File: conf/example_affiliations.json
Comment (aajarven) [reply conf/example_affiliations.json:1]: It's an example. It's a lot easier for someone to start using the tool if they have a "template" as opposed to just reading from docs that "the affiliations must be in a json file containing a list of dicts, each dict having keys 'first_name' whose value is the affected person's first name, 'last_name'..."

File: modifiers/lb_modifiers.py (hunk 1 above)
Comment (aajarven) [reply modifiers/lb_modifiers.py:102]: That's not in our hands: we decided to use Black for Python, and Black does what it pleases. If we want to change individual practices like that, we need to ditch Black altogether and switch to something else.

**Review by mmatthiesencsc — APPROVED** [review]

**Review by mmatthiesencsc — APPROVED** [review]

**Pull request merged**